tasks.go

```diff
@@ -56,6 +62,33 @@ func GetAllTasks(basepath string) ([]Task, error) {
 	return tasks, nil
 }
 
+// GetNagiosDataTask will return an array of Tasks each of which will dump the nagios data for one project.
+func GetNagiosTasks(projects []string, basepath string) ([]Task, error) {
+	var tasks []Task
+	var errors errorList
+	for _, p := range projects {
+		pods, err := getResourceNamesBySubstr(p, "pod", "nagios")
+		if err != nil {
+			errors = append(errors, err)
```

Comment:
Does it make sense to loop over pods when there was an error from `getResourceNamesBySubstr`?